Sources/Quick/Example.swift

```diff
@@ -71,7 +71,11 @@ final public class Example: NSObject {
         }
         group!.phase = .beforesFinished
 
-        closure()
+        do {
+            try closure()
+        } catch {
+            print("Test \(name) threw unexpected error: \(error.localizedDescription)")
```

Comment:
How to property report an error? I've tried `XCTFail`, but can't use the `callsite` information because of types (`StaticString` x `String`).

I wasn't able to get a reference to the `XCTestCase` that contains this example, but I think that would solve the issue as I could call [`recordFailure(withDescription:inFile:atLine:expected:)`](https://developer.apple.com/reference/xctest/xctestcase/1496269-recordfailure)